src/platform/testing/e2e/cypress/support/commands/webComponents.js

```diff
@@ -0,0 +1,246 @@
+const FORCE_OPTION = { force: true };
+
+Cypress.Commands.add('fillVaTextInput', (field, value) => {
+  if (value) {
+    const strValue = value.toString();
+    const element =
+      typeof field === 'string'
+        ? cy.get(`va-text-input[name="${field}"]`)
+        : cy.wrap(field);
+
+    element
+      .shadow()
+      .find('input')
+      .as('currentElement')
+      .type(strValue);
+
+    cy.get('@currentElement').should('have.value', strValue);
+  }
+});
+
+Cypress.Commands.add('fillVaTextarea', (field, value) => {
+  if (value) {
+    const strValue = value.toString();
+    const element =
+      typeof field === 'string'
+        ? cy.get(`va-textarea[name="${field}"]`)
+        : cy.wrap(field);
+
+    element
+      .shadow()
+      .find('textarea')
+      .as('currentElement')
+      .type(strValue);
+
+    cy.get('@currentElement').should('have.value', strValue);
+  }
+});
+
+Cypress.Commands.add('selectVaRadioOption', (field, value) => {
+  if (typeof value !== 'undefined') {
+    cy.get(`va-radio-option[name="${field}"][value="${value}"]`)
+      .as('currentElement')
+      .click();
+    cy.get('@currentElement').should('have.attr', 'checked');
+  }
+});
+
+Cypress.Commands.add('selectYesNoVaRadioOption', (field, value) => {
+  if (typeof value !== 'undefined') {
+    const selection = value ? 'Y' : 'N';
+    cy.selectVaRadioOption(field, selection);
+  }
+});
+
+Cypress.Commands.add('selectVaSelect', (field, value) => {
+  if (typeof value !== 'undefined') {
+    const element =
+      typeof field === 'string'
+        ? cy.get(`va-select[name="${field}"]`)
+        : cy.wrap(field);
+
+    element
+      .shadow()
+      .find('select')
+      .as('currentElement')
+      .select(value, FORCE_OPTION);
+    cy.get('@currentElement').should('have.value', value);
```

Comment:
Do we need to make this value into a string as well?

Reply:
good call, ill test this and make change if necessary